docs/content/statuses.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Component status
+---
+
+Each component is given a status that explains the component's maturity level.
+
+| Status         | Meaning                                                                |
+| -------------- |------------------------------------------------------------------------|
+| `Alpha`        | Experimental. Breaking changes are likely.                             |
+| `Beta`         | Usage encouraged, but not required.                                    |
+| `Stable`       | Usage expected.                                                        |
+| `Deprecated`   | Usage not allowed. May be removed at any time.                         |
+
+We expect many components to move from alpha to beta and from beta to stable
+soon. Feel free to watch for releases in the
+[repo](https://github.com/primer/view_components).
+
+You can also see component status at a glance on [this project
+board](https://github.com/primer/view_components/projects/3).
+
+## Criteria for changing component status
+
+### Promote from alpha to beta
+
+- The component does not use any deprecated `view_components` platform features (eg - Slots v1).
+- Documentation and storybook stories are complete.
+- The API is expected to be stable.
```

Comment:
Did we want to add a criteria for a certain % or # of internal uses?

Reply:
```suggestion
- The API is expected to be stable and the component is used in dotcom at least 3 times.
```

Something like this? I'm not sure we have a more specific but also broadly applicable at this point.

Reply:
I think this is fine for now 👍🏻